autoload/denops/server.vim

```diff
@@ -57,6 +57,7 @@ function! s:on_channel_opened(script, options, channel) abort
         \ 'script_args': ['--mode=' . s:engine, '--address=' . a:channel.address],
```

Comment:
Please modify `script_args` directly while `--inspect` does not work on Channel server thus it should not be a common option.

Reply:
I'm sure but I'm sorry to find no way to add script_args before `a:script` in [`denops/script.vim#L36`](https://github.com/vim-denops/denops.vim/blob/47de151b218f8a306d88e81eb0327721c6eb6ddd/autoload/denops/script.vim#L36) without editing `script.vim`. Is there any way to do that?

Reply:
I use `deno_args` instead of `script_args` because of `--inspect` does not work correctly after `a:script`, but I don't know using `g:denops#script#deno_args` in `server.vim` is correct because I'm beginner at viml. Please suggest change if this is not better way.

Reply:
:100:

Reply:
:100: